test/unit/browser/invalidation_mixin_test.dart

```diff
@@ -46,7 +46,7 @@ void main() {
       });
 
       test('calls validate, eventually', () {
-        Future onValidation = thing.invalidate();
+        Future<ValidationStatus> onValidation = thing.invalidate();
```

Comment:
explicit Type here for Dart 2 inference later on at `.then(expectAsync1((ValidationStatus status) {`